runtime/common/src/weights/mod.rs

```diff
@@ -15,7 +15,7 @@ pub mod system;
 pub mod timestamp;
 pub mod tokens;
 pub mod transaction_pause;
-pub mod treasury;
-pub mod utility;
+// pub mod treasury;
```

Comment:
It's necessary, but RN the files don't contain weights for some new methods. I didn't want to place there random values, so re-benchmarking is required.

Reply:
i would prefer to change the wieghts and add some weight. 

reason being, after rebenchmarking, it would not be needed to put this back - only weights will need to be updated. 

i have this sem-automated, and i would not need to remember and  make additional changes